### DISCUSSION TITLE
Enhancement: add checkpoints to practice mode

### DISCUSSION THREAD

**Discussion by FrankLoq** [FunkinCrew/Funkin#1639] [Ideas]
When you activate the practice mode that with one key you can make a checkpoint and with another one load it, I think it should be implemented because in difficult sections such as the milf drop beat it could be practiced without having to do the whole song until the section we want to practice
sorry for my bad english, is not my native language and i am using the translator

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa4** [comment]
This is a good idea

**Comment by aflacc** [comment]
You can't die in practice mode so whats the point

**Comment by FrankLoq** [comment]
> You can't die in practice mode so whats the point
the point is practice difficult sections without having to do the whole song until you get to that part
this would make the game less frustrating and more fun

**Comment by aflacc** [comment]
I'm pretty sure its not that easy to make a audio thing start at one point
